docs/content/standards/currency.mdx

```diff
@@ -35,56 +50,56 @@ The registry is a system-level shared object located at address `0xc`.
 <ImportContent source="crates/sui-framework/packages/sui-framework/sources/registries/coin_registry.move" mode="code" />
 </details>
 
-#### Core components
+### Core components
 
-**`CoinRegistry`:** The main system object that coordinates all currency-related operations. This shared object serves as the entry point for all registry operations and is created once during network initialization. It's address is `0xc`.
+**`CoinRegistry`:** The main system object that coordinates all coin-related operations. This shared object serves as the entry point for all registry operations and is created once during network initialization. Its address is `0xc`.
 
 **`Currency<T>`:** The core of the registry system, storing comprehensive information about each coin type including:
-- **Metadata management:** Basic currency information like name, symbol, decimals, description, and icon URL.
+- **Metadata management:** Basic coin information like name, symbol, decimals, description, and icon URL.
 - **Supply tracking:** Maintains supply state information (fixed, burn-only, or unknown).
 - **Regulatory status:** Tracks whether the coin is regulated with deny list capabilities.
-- **Capability references:** Links to `TreasuryCap`s and `MetadataCap`s for the coin type.
+- **Capability references:** Links to treasury and metadata capabilities for the coin type.
```

Comment:
Links to `TreasuryCap` - it was correct. It literally contains `TreasuryCap` and `MetadataCap` ID's. There's no concept of "treasury" without `TreasuryCap`.